mir_eval/melody.py

```diff
@@ -261,8 +274,11 @@ def resample_melody_series(times, frequencies, voicing,
         frequencies_resampled = scipy.interpolate.interp1d(times,
                                                            frequencies,
                                                            kind)(times_new)
+
     # Use nearest-neighbor for voicing if it was used for frequencies
-    if kind == 'nearest':
+    # if voicing is not binary, use linear interpolation
+    is_binary_voicing = np.array_equal(np.round(voicing), voicing)
```

Comment:
Is it any safer to do `np.all(np.logical_or(np.equal(voicing, 0), np.equal(voicing, 1)))`?

Reply:
yep agreed